taskEight.java

```diff
@@ -0,0 +1,10 @@
+// napisat' class kotoruy vuvodit v consol' vek iz polychenou datu (naprimer 873 god - eto 9-u vek, a 1701 god - eto 18-u vek)
+
+class taskEight {
```

Comment:
According to "Naming Conventions" you have to use first letter of each internal word of the class capitalized, please fix it in all classes

Reply:
fixed